epochStart/shardchain/trigger.go

```diff
@@ -263,13 +267,25 @@ func (t *trigger) checkIfTriggerCanBeActivated(hash string, metaHdr *block.MetaB
 			go t.requestHandler.RequestHeaderByNonce(sharding.MetachainShardId, nonce)
 		}
 	} else {
```

Comment:
can you do the check for the else condition then you would not need an if else construct

Reply:
done